raiden/ui/cli.py

```diff
@@ -92,6 +95,30 @@
         default=60,
         type=int,
     ),
+    click.option(
+        '--cli/--no-cli',
+        help=(
+            'Start with or without the command line interface. Defualt is to '
+            'start with the CLI disabled'
+        ),
+        default=False,
+    ),
+    click.option(
+        '--rpc/--no-rpc',
+        help=(
+            'Start with or without the RPC server. Default is to start '
+            'the RPC server'
+        ),
+        default=True,
+    ),
+    click.option(
+        '--port',
```

Comment:
Change the name of the option to be more specific. It should be `--api-port`. Even more so because we already have a `--port` command line option. Funny that `click` does not complain.